Source/Processors/RecordNode/BinaryFormat/BinaryRecording.cpp

```diff
@@ -28,6 +30,23 @@ String BinaryRecording::getProcessorString(const InfoObjectCommon* channelInfo)
 	return fName;
 }
 
+int BinaryRecording::getSamplesPerBlock()
+{
+	int blockSamps = MIN_SAMPLES_PER_BLOCK;
+
+	switch (m_blockSizeBoost)
+	{
+	case BINBLOCK_SMALL: break;
+	case BINBLOCK_MEDIUM: blockSamps *= 4; break;
+	case BINBLOCK_LARGE: blockSamps *= 16; break;
+	default: break;
+	}
+// FIXME - Diagnostics.
+std::cerr << "[BinaryRecording]  getSamplesPerBlock() returns: " << blockSamps << "\n";
```

Comment:
Change this to `LOGD`

Reply:
I've removed it instead (I'd missed it when removing other debugging tattles).